src/errors.md

```diff
@@ -0,0 +1,128 @@
+# Common errors
+
+## Command Line Interface
+
+[CLI documentation](https://github.com/SubstraFoundation/substra/blob/master/references/cli.md#summary)
+
+> Remember: you can use `substra --help` and `substra <command> --help` anytime!
+
+### substra config
+
+> `substra config --profile <profile> <url>`
+
```

Comment:
We may remove these lines since we don't provide any errors for this command?

Reply:
In fact, you get the `HTTPConnectionPool` error if you don't use the `--profile` param, that's why I plced this reminder. what do you think about it?

Reply:
I thought the 503 was for the substra login only. (maybe the rendering will be different on the doc website)

Reply:
Actually both the 503 error and the HTTPConnectionPool error can be encountered with any CLI command?

Reply:
Yes, but I saw more frequently on the first steps.